api/routers/sessions.py

```
@@ -0,0 +1,84 @@
+    )
+    return SessionsOut(
+        active=[ssh_session_out(s) for s in active_result.scalars().all()],
+        recent=[ssh_session_out(s) for s in recent_result.scalars().all()],
+    )
+
+
+@router.post("/{session_id}/terminate")
+async def terminate_session(
+    request: Request,
+    session_id: str,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)